Makefile

```diff
@@ -9,7 +9,9 @@ run:
 
 create-migration:
 	@read -p "migration name: " name; \
-	migrate create -ext sql -dir internal/database/postgres/migration -seq $$name
+	timestamp=$$(date +%Y%m%d%H%M); \
+	touch internal/database/postgres/migration/$${timestamp}_$$name.up.sql; \
+	touch internal/database/postgres/migration/$${timestamp}_$$name.down.sql
```

Comment:
I don't think we should rename already "executed" scripts, because they're already versioned and timestamps should go sequentially after the 000001, 000002, etc, so there should be no conflicts.